src/programQueueProcessor/unit.test.ts

```diff
@@ -256,42 +256,44 @@ describe("kafka integration", () => {
       expect(test_ca_documents).to.equal(testDonorIds.length);
 
       // check if new rdpc data is relfected in TEST-CA
-      testDonorIds.forEach(async (donorId) => {
-        const esQuery = esb
-          .requestBodySearch()
-          .size(testDonorIds.length)
-          .query(esb.termQuery("donorId", donorId));
+      async () => {
```

Comment:
why this async function? it's also not called so I don't think it's reaching the loop